build.sbt

```diff
@@ -198,5 +198,28 @@ lazy val docs = project
       "mdoc" -> url("https://scalameta.org/mdoc/"),
       "Laika" -> url("https://planet42.github.io/Laika/"),
       "sbt-unidoc" -> url("https://github.com/sbt/sbt-unidoc")
-    )
+    ),
+    mdocVariables ++= {
+      import coursier.complete.Complete
+      import java.time._
+      import scala.concurrent._
+      import scala.concurrent.duration._
+      import scala.concurrent.ExecutionContext.Implicits._
+
+      val startYear = YearMonth.now().getYear.toString
+
+      val sjsVersionFuture =
+        Complete().withInput(s"org.scala-js:scalajs-library_2.13:").complete().future()
```

Comment:
I spun in circles for a bit trying to find the latest version via the sbt-scalajs artifact until I remembered this issue exists:
- https://github.com/coursier/coursier/issues/2178

which of course is because of:
- https://github.com/sbt/sbt/issues/3410